src/data/equiv/basic.lean

```diff
@@ -2263,6 +2263,13 @@ protected def congr {ra : α → α → Prop} {rb : β → β → Prop} (e : α
   left_inv := by { rintros ⟨a⟩, dunfold quot.map, simp only [equiv.symm_apply_apply] },
   right_inv := by { rintros ⟨a⟩, dunfold quot.map, simp only [equiv.apply_symm_apply] } }
 
+/-- A lemma for simplifying the equivalence between quotient spaces as stated just above -/
+@[simp]
+lemma congr_map_simp {α : Sort u} {β : Sort v} {ra : α → α → Prop} {rb : β → β → Prop} (e : α ≃ β)
+  (eq : ∀ (a₁ a₂ : α), ra a₁ a₂ ↔ rb (e a₁) (e a₂)) (a : α) :
+  (quot.congr e eq).to_fun (quot.mk ra a) = quot.mk rb (e a)
+  := by unfold quot.congr quot.map
```

Comment:
This lemma is a good idea, but:
* the name should be `congr_mk`
* you don't need to repeat the variables that are already available in `variables`
* you shouldn't use `to_fun`
* the proof is just `rfl`!

```suggestion
@[simp]
lemma congr_mk {ra : α → α → Prop} {rb : β → β → Prop} (e : α ≃ β)
  (eq : ∀ (a₁ a₂ : α), ra a₁ a₂ ↔ rb (e a₁) (e a₂)) (a : α) :
  quot.congr e eq (quot.mk ra a) = quot.mk rb (e a) := rfl
```

Reply:
Can you add the same lemma for `quotient.congr` below?

Reply:
I recommend you add the `quotient.congr` one in this PR, else you'll end up having to wait for two long builds before you have both lemmas. The ring and module ones are fine to leave to a separate PR, as they're not in so fundamental a file

Reply:
Sure ! I'll add them now

Reply:
Is there an easy way of going back and forth between the changes made on GitHub and those made on VSCode ? I'm not sure how to implement the changes you made and them add my extra code without making a new PR.